src/docs/zksync-era.mdx

```diff
@@ -105,52 +105,50 @@ links:
+    | FA | STATICCALL | | Executes a static message call into an account.<br /><br />Memory growth occurs after the call has ended, leading to a difference in the memory size (`msize`). As a result, ZKsync Era may not panic in situations where EVM would panic due to differences in memory growth. | Executes a static message call into an account.<br /><br />Memory growth occurs before the call itself. <Modified /> |
 
-    Additional information on the differences in OPCODES can be in [Differences with Ethereum Docs](https://era.zksync.io/docs/reference/architecture/differences-with-ethereum.html#codecopy).
+    Additional information on the differences in OPCODES can be found in [the ZKSync protocol documentation](https://docs.zksync.io/zksync-protocol/differences/evm-instructions).
 
 </Section>
 
 <Section title="Precompiled Contracts">
 
     <Legend />
 
+    > ⚠️ Precomiles `ecAdd`, `ecMul`, and `ecPairing` are supported on ZKsync Era.
+
     The following Precompiled Contracts behave differently compared to the canonical Ethereum L1
 
     | Address | Name | Rollup Behaviour | Ethereum L1 Behaviour |
     | :----- | :----- | :--------------- | :-------------------- |
     | <Copy label="0x03" value="0x03" /> | `RIPEMD-160` | Not supported | Hash function <Unsupported /> |
     | <Copy label="0x04" value="0x04" /> | `identity` | Not supported | Returns the input <Unsupported /> |
     | <Copy label="0x05" value="0x05" /> | `modexp` | Not supported | Arbitrary-precision exponentiation under modulo <Unsupported /> |
-    | <Copy label="0x06" value="0x06" /> | `ecAdd` | Not supported | Point addition (ADD) on the elliptic curve 'alt_bn128' <Unsupported /> |
-    | <Copy label="0x07" value="0x07" /> | `ecMul` | Not supported | Scalar multiplication (MUL) on the elliptic curve 'alt_bn128' <Unsupported /> |
-    | <Copy label="0x08" value="0x08" /> | `ecPairing` | Not supported | Bilinear function on groups on the elliptic curve 'alt_bn128' <Unsupported /> |
     | <Copy label="0x09" value="0x09" /> | `blake2f` | Not supported | Compression function F used in the BLAKE2 cryptographic hashing algorithm <Unsupported /> |
     | <Copy label="0x0a" value="0x0a" /> | `pointEvaluation` | Not supported | Verifies a KZG proof <Unsupported /> |
+    | <Copy label="0x100" value="0x0a" /> | `Secp256r1` | Supported | Efficient elliptic curve cryptography for digital signatures on mobile devices and hardware that have a TEE <Added /> |
```

Comment:
Can you rename the precompile to "p256Verify" and change the description to be the same as on [Optimism](https://www.rollup.codes/optimism#precompiled-contracts)?